docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md

```diff
@@ -912,4 +912,3 @@ We’re sorry, but we don’t have any specific information about this error. Yo
  [C# Programming Guide](../../csharp/programming-guide/index.md)  
  [C# Reference](../../csharp/language-reference/index.md)  
  [Talk to Us](/visualstudio/ide/talk-to-us)  
```

Comment:
This link is broken, @pkulikov. It should be /visualstudio/ide/talk-to-us.md.

Reply:
@rpetrusha it's not broken: if I open [the current version of the page](https://docs.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error) and click the link, I get to [the intended place](https://docs.microsoft.com/en-us/visualstudio/ide/talk-to-us).

In general, such links—that point to the `docs.microsoft.com` pages that are not in this repo—do not demand '.md' suffix. I think such a suffix even would break a link.

Reply:
It's correct like this. Cross-repo links don't have the .md extension.

Reply:
I completely forgot about the cross-repo linking. Sorry, @pkulikov and @mairaw.